code/game/machinery/defibcharger.dm

```diff
@@ -87,29 +87,15 @@ obj/machinery/recharger/defibcharger/wallcharger/process()
 	return -1
 
 obj/machinery/recharger/defibcharger/wallcharger/attackby(obj/item/weapon/G as obj, mob/user as mob)
-	if(..())
-		return
-	if(istype(user,/mob/living/silicon))
-		return
 	if(istype(G, /obj/item/weapon/melee/defibrillator))
+		if(..())
+			return
 		var/obj/item/weapon/melee/defibrillator/D = G
```

Comment:
All good, little niggle though, you don't actually drop the defib anymore since its only handled in the parent for guns